CHANGES.rst

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+Version 0.2.2 [2020-11-19]
+--------------------------
+
+Bugfixes
+~~~~~~~~
+
+- Fixed notification alert sound being played from multiple windows
```

Comment:
looks like this is missing 2e4b39cb4b1b7b20ee11ec2158e451e715aa2f03

Reply:
This fix only works for the test project. It does not applies to module due to the bug mentioned in the PR.